bench/tiered/push_pull.c

```diff
@@ -311,23 +329,24 @@ run_test(const char *home, uint32_t num_records, bool flush, int counter)
  *     Populate the table.
  */
 static void
-populate(WT_SESSION *session, uint32_t num_records)
+populate(WT_SESSION *session, uint32_t num_records, uint32_t counter)
 {
     WT_CURSOR *cursor;
-    uint64_t i, str_len;
+    WT_ITEM item;
+    uint64_t v, i;
 
-    srand((uint32_t)getpid() + num_records);
+    /* Seed the random number generator */
+    v = (uint32_t)getpid() + num_records + (2 * counter);
+    __wt_random_init_custom_seed(&rnd, v);
 
-    str_len = sizeof(data_str) / sizeof(data_str[0]);
-    for (i = 0; i < str_len - 1; i++)
-        data_str[i] = 'a' + (uint32_t)rand() % 26;
-
-    data_str[str_len - 1] = '\0';
+    fill_random_data();
```

Comment:
Move this into the loop as mentioned before.

Reply:
1ac8107d21a11c5cb634b31bcedca81374709d70